examples/inference/README.md

```diff
@@ -0,0 +1,125 @@
+# Colossal-Inference
+
+
+## Table of Contents
+
+-   📚 Introduction
+
+- 🔨 Installation
+- 🚀 Quick Start
+- 💡Usage
```

Comment:
There is an empty line between introduction and installation.

Reply:
Make them clickable by using something like

- [📚 Introduction](#📚-introduction)

You can test it on your own.

Reply:
Do the same for the development doc as well.

Reply:
Fixed.